packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/TabPanels/PaymentTabPanel/UpdatePDF/UpdatePDFSection.tsx

```diff
@@ -0,0 +1,46 @@
+import { components } from '@/api';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import { Box, Typography } from '@mui/material';
+
+import { UpdatePDFModal } from './UpdatePDFModal';
+
+type Props = {
+  invoice: components['schemas']['InvoiceResponsePayload'];
+};
```

Comment:
Please avoid creating a separate type if it’s only used once, as it adds an unnecessary wrapper. Use components['schemas']['InvoiceResponsePayload'] directly instead